src/q5/stat-FSAB1105/exam/2016/Août/All/stat-FSAB1105-exam-2016-Août-All.tex

```diff
@@ -0,0 +1,229 @@
+\documentclass[fr]{../../../../../../eplexam}
+
+\hypertitle{Probabilité et Statistiques}{5}{FSAB}{1105}{2016}{Août}{All}
+{BAC3 2015-2016}
+{Anouar El Ghouch}
+
+\section{/6}
+
+\begin{enumerate}
+\item Pour les Jeux Olympiques, on a besoin de 200 balles. La firme A en produit 120 avec un taux de défaillance de 5\% et la firme B produit les 80 restantes avec un taux de défaillance de 10\%. Lors d'un match, 3 balles se révèlent être défectueuses.\\
```

Comment:
Les \\ en bout de ligne... 🤢

Reply:
:sob:

Reply:
Si tu veux faire un paragraphe, tu mets quoi alors en fait? :smile:

Reply:
Si tu(*) veux faire un paragraphe, tu mets simplement un double retour à la ligne.
Simple.
Si tu veux juste supprimer l'alinéa, il y a des trucs comme `\noindent`.
Si tu veux réduire l'interligne, il y a `\setlength{\parskip}{1em}`.
Il y a suffisamment d'alternatives pour toutes les utilités sémantiques de ce `\\` affreux que pour qu'on n'ait jamais à le mettre dans du texte.
Et en plus, ils osent mettre `\\` ET mettre un double retour à la ligne. Super. En plus de la mocheté du code source, de la mocheté du rendu, et de la mocheté des connaissances de LaTeX, ça rajoute un "underful \hbox".
Si un jour j'ai la possibilité de créer des lois, je rendrai l'utilisation de `\\` à la liste des crimes contre l'humanité.
(*) Je m'adresse à un utilisateur de LaTeX, pas à toi ;)

Reply:
Il me semble que parfois le double retour à la ligne ne fonctionne pas, par exemple après un itemize...

Reply:
Je serais curieux de voir un exemple où ça ne marche pas ;)
Après un itemize, on est toujours dans le paragraphe courant, et pour en sortir il faut encore un retour à la ligne. En tout cas c'est ce que je peux tester.

Reply:
Peut-être que c'est la compilation du package sur `Atom` qui n'est pas parfaite, il faudrait que j'essaie avec le `make`.

Reply:
OK, mais tu mets quoi si tu veux un retour à la ligne avec un espace en les 2 lignes, comme ce que fait `\\`? C'est quand même mieux d'avoir 2 possibilités, un retour à la ligne et un retour à la ligne avec espace entre les lignes non?
![image](https://user-images.githubusercontent.com/25173236/44450963-9bd69480-a5f2-11e8-83e4-8597ab4caac3.png)

Reply:
Tu changes l'espace inter-paragraphe juste pour ce paragraphe. Par exemple en mettant `\setlength{\parskip}{\baselineskip}` juste avant le paragraphe qui doit être plus espacé (donc là où on met le `\\`) et en encadrant cette commande et ce paragraphe entre accolades (pour limiter l'effet à cette région du document). Par exemple
```
The encoding has traditionally been either ASCII, sometimes EBCDIC. Unicode-based encodings such as UTF-8 and UTF-16 are gradually replacing the older.\\
Files that contain markup or other meta-data are generally considered plain-text, as long as the entirety remains in human-readable form (as in ).

{\setlength{\parskip}{\baselineskip}
Files that contain markup or other meta-data are generally considered plain-text, as long as the entirety remains in human-readable form (as in ).
}

Files that contain markup or other meta-data are generally considered plain-text, as long as the entirety remains in human-readable form (as in ).
```
Quand tu y penses, c'est ce que tu veux faire au final ? Mettre plus d'espace entre deux paragraphes.
Ce que je n'aime pas avec `\\` suivi d'un retour à la ligne c'est que ça provoque un "Underfull \hbox (badness 10000) in paragraph at lines 11--13" (ce sont les lignes du paragraphe). En effet, on dit à LaTeX qu'on va revenir à la ligne, il se prépare à écrire du texte, et puis on lui dit qu'en fait non, il n'y aura pas de texte. Une badness de 10000 n'est pas anodine.

Reply:
Je vois... J'utilerais au final uniquement le double retour à la ligne et les commandes `\smallbreak`, `\medbreak`, `\bigbreak`, elles sont plus courtes pour le même effet.

Reply:
Je ne connaissais pas ces commandes. Je retiendrai ;)
Tout est mieux que `\\ \n\n` de toutes façons ;)

Reply:
Mince, j'ai oublié de passer lien :smile: : 
https://tex.stackexchange.com/a/136953